ui/core/proto_utils/action_id.ts

```diff
@@ -747,6 +749,16 @@ const petNameToActionId: Record<string, ActionId> = {
 	Ghoul: ActionId.fromSpellId(46584),
 	'Army of the Dead': ActionId.fromSpellId(42650),
 	Valkyr: ActionId.fromSpellId(71844),
+	Felguard: ActionId.fromSpellId(30146),
+	Felhunter: ActionId.fromSpellId(691),
+	'Ebon Imp': ActionId.fromSpellId(688),
+	Imp: ActionId.fromSpellId(688),
+	Succubus: ActionId.fromSpellId(712),
+	Doomguard: ActionId.fromSpellId(18540),
+	'Summon Doomguard': ActionId.fromSpellId(18540),
+	Infernal: ActionId.fromSpellId(1122),
```

Comment:
The aim is to do it like WCL does it:
- Have the summoning of the pet as a cast metric on the parent row (including the hit/tick damage averages)
- Have the child rows (abilities) have the actual spell counts and hits etc

Reply:
Huh? WCL doesn't use the summoning spell either, they use the pet name:
![image](https://github.com/user-attachments/assets/4240f72a-8738-4d6e-8b6e-ace78ed3dd3d)

Obviously we can't mirror that exactly since we don't have a specific pet, but the generic name like `Felhunter` as a replacement makes a lot more sense then `Summon Felhunter`.

Reply:
@Tharre You are right, reverting this. Not sure where I found this 🤔